Source/Scene/Model.js

```diff
@@ -3293,6 +3306,37 @@ define([
         };
     }
 
+    function createClippingPlanesLengthFunction(model) {
+        return function() {
+            return model.clippingPlanes.length;
+        };
+    }
+
+    var scratchCartesian = new Cartesian3();
+    var scratchMatrix = new Matrix4();
+
+    function createClippingPlanesFunction(model, context) {
+        return function() {
+            var scale = getScale(model, context.uniformState.frameState);
```

Comment:
Do we really need to do all this math each frame?  Can we protect this with a dirty flag?